### ISSUE TITLE
Session is not registered. Closing connection

### ISSUE THREAD

**Issue by jimmyathf** [mrniko/netty-socketio#460]
While doing performance Test, I got error like " [ntLoopGroup-3-3] c.c.socketio.transport.PollingTransport  : 4982e004-8d8b-479c-8fdd-e1c918668d23 is not registered. Closing connection "．And only 2000 clients are connetced.  If I set UpgradeTimeOut with 180000, no such error  will happen even 10000 clients are connected. Seems got IO stress with PING and PONG. Any suggestions will be appreciated.

**Comment by lovebing** [comment]
I got the same error, but increasing the value of UpgradeTimeOut didn't work.

**Comment by song2song** [comment]
Multiple socketio server, do not set up a connection with a polling, specify a websocket transport，Because the polling HTTP request will be requested to a different machine, causing sid is not registered.

**Comment by pigercc** [comment]
> Multiple socketio server, do not set up a connection with a polling, specify a websocket transport，Because the polling HTTP request will be requested to a different machine, causing sid is not registered.

you are right, front end socketio js set websocket only , can solve this problem
```
transports:['websocket']
```

**Comment by SKOTYK** [comment]
@song2song  thanks a lot, polling and round robin LB is not a good idea :D

**Comment by oryjk** [comment]
> Multiple socketio server, do not set up a connection with a polling, specify a websocket transport，Because the polling HTTP request will be requested to a different machine, causing sid is not registered.

Why polling request will be requested to a different machine?
